### ISSUE TITLE
Typing Unicode Hex switches to Search box

### ISSUE THREAD

**Issue by bvn9ne** [pbek/QOwnNotes#2696]
#### Expected behaviour
Typing in unicode character as hex such as holding alt and typing: +, 3, 0, F, 0 will keep cursor at current location.

#### Actual behaviour
Typing in unicode character as hex such as holding alt and typing: +, 3, 0, F, 0 will switch to the note search box. If you click back into the note before releasing alt will type the correct character, in this case ヰ.

#### Output from the debug section in the settings dialog

<details><summary>Expand</summary>

</details>

#### Relevant log output in the Log panel

<details><summary>Expand</summary>

</details>

**Comment by pbek** [comment]
I guess you just hit a keyboard shortcut, some of them use the Alt key. You can overwrite those in the shortcut settings.

**Comment by bvn9ne** [comment]
I did (alt+f). I just didn't know if it would be possible for alt followed by the + sign to catch the input so it doesn't change focus. It's not the end of the world, I just use a lot of alt-code unicode when typing so it seemed better in the long run than remapping.On Jan 6, 2023 3:54 PM, Patrizio Bekerle ***@***.***> wrote:
I guess you just hit a keyboard shortcut, some of them use the Alt key. You can overwrite those in the shortcut settings.

**Comment by pbek** [comment]
I don't know if Qt's QPlainTextEdit does support that... Kate doesn't support it neither, so I guess not. I haven't seen such shortcuts anywhere yet... Where do they work for example and on which platform?

**Comment by bvn9ne** [comment]
The only example I tested was Notepad++ on Windows. That's how I noticed the difference. I know many things don't, such as browsers. I can test LibreOffice and MS office later. I am willing to guess that they would do the same.On Jan 7, 2023 5:24 AM, Patrizio Bekerle ***@***.***> wrote:
I don't know if Qt's QPlainTextEdit does support that... Kate doesn't support it neither, so I guess not. I haven't seen such shortcuts anywhere yet... Where do they work for example and on which platform?
